src/components/AMMPrices/TokenAMMPriceRow.tsx

```diff
@@ -82,7 +88,7 @@ export default function TokenAMMPriceRow({
             overlayInnerStyle={{ ...fontStyle }}
           >
             <ExternalLink href={exchangeLink} style={{ fontWeight: 400 }}>
-              {`${formattedNum(WETHPrice)} ${tokenSymbol}/1 ETH`}
+              {`${preparePrice(WETHPrice)} ${tokenSymbol}/1 ETH`}
```

Comment:
**Nit:** think we could just name this `formatPrice` for consistency

Reply:
agreed!